insert-ordered-containers.cabal

```diff
@@ -46,6 +43,8 @@ library
     , base-compat           >=0.10.5  && <0.12
```

Comment:
- `base` should be bumped to `>=4.9`
- also `transformers` lower bump can be bumped and CPP cleaned up. Dropping old GHC support should be fine by now.

`swagger2` would be affected, but it should be fine https://github.com/GetShopTV/swagger2/issues/199


I'll do that after merge & before release

Reply:
Alright. I also have a corresponding swagger2 PR that adds support for optics btw, that's what prompted this PR ;)

Reply:
I'm curious to see swagger2 PR.

Reply:
Too fast enter.

I'll make a release today.

Does `swagger2` change introduce `LabelOptic` instances for everything, so you don't have problems with name clashes? I.e. `optics` usage is 

```haskell
mempty
   & #type .~ SwaggerString
```

?

If it's so, than I'll support the change. It's quite neat that way.

Reply:
Yes, that's exactly how it looks.